cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -1,6 +1,10 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "$defs": {
+    "eol_bionic": {
+      "deprecated": true,
+      "deprecated.time": "EOL Ubuntu bionic"
+    },
```

Comment:
I have changed its value to `Dropped after April 2027.` (End of standard support of Jammy).

TL;DR I dropped `deprecated.time` and include the extra info in `description`.

Regarding `deprecated.time`, I have thought about it and I came to the conclusion that it is not really needed.

I was solving two problems:

1. Using it as a kind of variable by referencing it in other sub-schemas, in order to avoid repetition.
2. Supplementing the deprecation description.

(1.) implied to merge `deprecated.time` and `description` within code.

Thinking deeper about it:

* The lack of variables, string concatenation, etc in `jsonschema` should not enforce this added complexity
* The moment we add this new key to the schema, we must maintain it to keep backwards compatibility, as we don't know if clients would be coupled to it.
* If the schema repetition / complexity increases a lot, then this problem sounds as more soluble by using a templating engine (`as Jinja2`).

I decided to KISS and add the deprecation date info to `description`.